app/forms/renewal_received_form.rb

```diff
@@ -10,4 +10,11 @@ def initialize(transient_registration)
 
   # Override BaseForm method as users shouldn't be able to submit this form
   def submit; end
+
+  def dashboard_link(current_user)
+    return unless current_user.present?
+    id = current_user.id
+    root = Rails.configuration.wcrs_frontend_url
+    "#{root}/user/#{id}/registrations?locale=en"
```

Comment:
Don't actually need the `?locale=en" on the end so I'd be minded to leave it off.